src/engine/sys/con_tty.cpp

```diff
@@ -85,9 +85,9 @@ void WriteToStdout(const char* text) {
 
 	if (ret < 0) {
 		// Do not log to stdout
-		Log::Dispatch( Str::Format("Error writing to the terminal: %d", errno),
+		Log::Dispatch( Str::Format("Error writing to the terminal: %s", strerror(errno)),
```

Comment:
let's print the code and the string?

Reply:
The standard in Daemon code seems to be to print only `strerror(errno)`, and not the number. I guess even worst case if the message is translated it's often clear which one it is. If that's a problem though, then we should rework more than just this line